Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -166,6 +167,64 @@ final class GoogleSheetsManager: ObservableObject {
     NotificationCenter.default.post(name: .hasSheetInDefaults, object: nil, userInfo: [Notification.Name.hasSheetInDefaults: file])
   }
   
+  private func createSheetsValueRangeFrom(amount: String,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/e6c795dd72b19b851d3b62f7797c3a34894b15f1/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L170"></span>Function should have 5 parameters or less: it currently has 6<br />
<code>function_parameter_count</code> <code>GoogleSheetsManager.swift:170</code></td>
    </tr>
  </tbody>
</table>